build: Error when bulding a package in depends for HOST=x86_64-apple-darwin16

On master (8e0f341779e1d2754432765ec09deb7488f5b093):
```
$ make -C depends qt HOST=x86_64-apple-darwin16
make: Entering directory '/home/hebasto/guix/GitHub/bitcoin/depends'
Fetching zlib-1.2.11.tar.gz from https://www.zlib.net
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  593k  100  593k    0     0   411k      0  0:00:01  0:00:01 --:--:--  411k
/home/hebasto/guix/GitHub/bitcoin/depends/work/download/zlib-1.2.11/zlib-1.2.11.tar.gz.temp: OK
Extracting zlib...
/home/hebasto/guix/GitHub/bitcoin/depends/sources/zlib-1.2.11.tar.gz: OK
Preprocessing zlib...
Configuring zlib...
tar: /home/hebasto/guix/GitHub/bitcoin/depends/built/x86_64-apple-darwin16/native_cctools/native_cctools-55562e4073dea0fbfd0b20e0bf69ffe6390c7f97-ee52008b72b.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
make: *** [funcs.mk:261: /home/hebasto/guix/GitHub/bitcoin/depends/work/build/x86_64-apple-darwin16/zlib/1.2.11-56aa350339c/./.stamp_configured] Error 2
make: Leaving directory '/home/hebasto/guix/GitHub/bitcoin/depends
```

--- a/depends/funcs.mk
+++ b/depends/funcs.mk
@@ -265,4 +265,4 @@ $(foreach package,$(all_packages),$(eval $(call int_config_attach_build_config,$
 $(foreach package,$(all_packages),$(eval $(call int_add_cmds,$(package))))
 
 #special exception: if a toolchain package exists, all non-native packages depend on it
-$(foreach package,$(packages),$(eval $($(package)_unpacked): |$($($(host_arch)_$(host_os)_native_toolchain)_cached) $($($(host_arch)_$(host_os)_native_binutils)_cached) ))
+$(foreach package,$(packages),$(eval $($(package)_extracted): |$($($(host_arch)_$(host_os)_native_toolchain)_cached) $($($(host_arch)_$(host_os)_native_binutils)_cached) ))

